### ISSUE TITLE
Conference News Policy

### ISSUE THREAD

**Issue by kjaymiller** [Python-Community-News/Topics#25]
We have a bunch of potential updates to how we report on conferences to make the show more inclusive and open to global contributors.

We are hoping to make it easier for people to tell us about their upcoming conference. Before we can put this in place there should be a discussion on how we qualify conference submissions.

## Things that need to be addressed.

- [ ] What are the **qualifications** for Conferences that should be submitted?
- [ ] **How will people submit** their Conference Submissions?
- [ ] **What is our (the hosts) role** in this process and managing conference announcements?

# Draft Conference Section
Below is my draft of providing solutions for these questions

### Qualifications of Conferences

Conferences should meet one or more of the following requirements:

1. The Conference **Celebrates Python in a Community or Discipline** (PyCon US, PyGotham, PyData)
2. The Conference **Celebrates a Community where Python is widely used** (TableauCon, Open Data Science Conference)
3. The Conference is a General Technology Focus with a **Python/Data Science specific track** (All Things Open)
4. The Conference is a **Not for Profit General Technology Focus Conference**(!!Con)
5. Any Conference that is being **sponsored by Python Community News or One of its hosts will be attending as a speaker** (Not through sponsorship of an employer)

> NOTE: All Conference Submissions will be vetted. **Hosts have the final approval**

Furthermore Conferences must have the following
- Public CFP (The CFP window may be closed but a CFP process must have taken place)
- Code of Conduct
- Either be a New Conference or have a History of conference Conduct that are in line with the Python Community News' Code of Conduct

### Types of Announcements

The show will report on the following announcements:
 1. Conference Dates Announced
 2. Tickets Go On Sale (We will not do any follow up posts regarding discount windows)
 3. CFP Opens
 4. Last Call for CFP (1 Week Out)

We will post as soon as we are made aware of the information. We will mention if a deadline has passed. We will not announce a conference after the ticket sale window has closed.

### Conference Announcement Submission Process

Conferences can be submitted in one of the following ways:
1. email to conferences@pythoncommuntynews.com
2. @mentioning the conference announcement on twitter ([@pyCommunitynews](https://twitter.com/pycommunitynews))
3. DM the [@PyCommunityNews Twitter](https://twitter.com/pycommunitynews) with the relevant information.
3. Filling out conference issue template

All Conference announcements should have an issue made. Issues will be closed and not reported on if they fail to meet the criteria for a valid conference, announcement type, or are not approved but the hosts.

### Video Contributions
One of the additions to the conference section is the addition of Video Submissions from the Conference organizers. 

We ask that video submissions only be submitted by one of the conference organizers and have as much of the information below. We will only air one video per conference event.

1. Where the event is being held. We would also love some information about the community there.
2. Any Python Communities Participating in the Event (PyLadies, DjangoGirls, etc)
3. CFP Dates
4. Ticket Sell Dates
5. Whether the event will be online, in-person or both (And Costs of Tickets for each one)

Please submit the video via unlisted YouTube video to help with this process.
We will play the video through Youtube. To allow for their subtitles. Subtitles will be in English but the video can be in any language the speaker is comfortable in as long as YouTube can provide a translation.

**Comment by kjaymiller** [comment]
@jonafato if approved I would suggest we either add it to a wiki section and link to it via the README OR Add it directly to the README. (I would prefer the former).

**Comment by jonafato** [comment]
(I'll preface this by saying that an issue comment doesn't allow for a great in-line commenting experience.) Overall +1 on having documentation around this, though I think we can start with a shorter version here. We also covered parts of this previously in https://github.com/kjaymiller/Python-Community-News/issues/35. Some thoughts below.

> ### Qualifications of Conferences

This is another one of those cases where I'm going to push for "anything of interest to the Python community". We're enumerating a bunch of rules and then tacking on a statement that says you and I have final decision over what goes into the show at the end of the day. I'm good with listing examples, as they might highlight conferences that people wouldn't think to tell us about, but trying to enumerate rules for something that's ultimately fuzzy makes the whole thing more confusing IMHO.

If we go the "anything of interest" route, I think this comes down to three points that are our responsibility, not the responsibility of anyone sharing a conference with us:
1. If it's obvious, it's obvious. No one will be surprised by us covering PyCon US news on the show, and vice versa for some conference that's not Python or even tech related as a rule.
2. If it's not obvious, and we decide to include it, part of our job in covering it is to explain why it's relevant to our listeners and readers. i.e. we should answer "how does a civil engineering conference relate to the Python community?" if we discuss it.
3. If we don't see how it ties back to Python up front, we can always ask someone who shared the topic for more details.

One specific point I'd push back on is the latter half of this:
> Any Conference that is being sponsored by Python Community News or One of its hosts will be attending as a speaker (Not through sponsorship of an employer)

I don't think that you or me speaking at a conference makes it inherently relevant for PCN, and it shouldn't be a special case.

---

> ### Types of Announcements
:+1: This is all in line with what we discussed in https://github.com/kjaymiller/Python-Community-News/issues/35.

---

> ### Conference Announcement Submission Process
I think there are too many options listed here, which will lead to things slipping through the cracks and make the conference roundup much harder to build automation around. Everything else about the show goes solely through the GitHub issues workflow, and conferences should as well. (I've expressed this before, but ideally this isn't even necessary and we can pull conference data from existing structured data sources.)

---

> ### Video Contributions
Can we leave this out until we have an example to point to? This is a higher-touch option that I assume we'll need to ask for directly, and I don't want to see someone spend a lot of time on a 20 minute video when we asked for a 20 second one.
